summingbird-storm/src/main/scala/com/twitter/summingbird/storm/Edges.scala

```diff
@@ -0,0 +1,101 @@
+package com.twitter.summingbird.storm
+
+import com.twitter.bijection.{ Injection, Inversion }
+import com.twitter.summingbird.storm.builder.{ EdgeGrouping, OutputFormat, Topology }
+import scala.util.Try
+import java.util.{ ArrayList => JAList, List => JList }
+import StormTopologyBuilder._
+
+/**
+ * This companion object contains all `Edge`s types used by Summingbird's storm topology.
+ */
+private[storm] object Edges {
+  def shuffleItemToItem[T](
+    sourceId: Topology.EmittingId[Item[T]],
+    destId: Topology.ReceivingId[Item[T]],
+    withLocal: Boolean
+  ): Topology.Edge[Item[T], Item[T]] = Topology.Edge(
+    sourceId,
+    EdgeFormats.item[T],
+    if (withLocal) EdgeGrouping.LocalOrShuffle else EdgeGrouping.Shuffle,
+    identity,
+    destId
+  )
+
+  def groupedAggregatedToSummer[K, V](
```

Comment:
Doc comments for all public methods here. We should make clear the terminology around Aggregated, Sharded and Item, perhaps by pointing to the right place to learn this use.